wikimedia/wikipedia-ios
# Switch Notifications Center Inbox View to multi-select component

**Phabricator:** 
https://phabricator.wikimedia.org/T343266

### Notes
This PR adjusts the `NotificationsCenterInboxView` to internally use the new `WKFormView` component, so that it can display toggles.

This PR is dependent on https://github.com/wikimedia/wikipedia-ios/pull/4596.

### Test Steps
1. Go to Notifications Center Inbox View.
2. Confirm you now see toggles instead of checkmarks. Confirm selecting toggles continues to filter the Notifications Center view.
3. Terminate and relaunch. Confirm toggles you had previously selected display as before (i.e. persistence works correctly).
